backend/src/components/nmfApplication.js

```diff
@@ -19,22 +19,33 @@ function mapNMFApplicationObjectForBack(data) {
   data.remoteCommunities = data.remoteCommunities ? ((data.remoteCommunities === "Yes") ? 1 : 0) : null;
   return new MappableObjectForBack(data, NMFApplicationMappings).toJSON();
 }
-  
-async function getNMFApplication(req, res) {
-  let query = `ccof_rfi_pfi_nmfs?$filter=(_ccof_applicationccfri_value eq ${req.params.ccfriId})`;
+
+async function getNmfApplicationByCcfriId(ccfriId) {
+  let query = `ccof_rfi_pfi_nmfs?$filter=(_ccof_applicationccfri_value eq ${ccfriId})`;
```

Comment:
can also be const

Reply:
Funny how many of these get a light shone on them when they're copied and pasted from existing code xD

Reply:
Right!! A lot of these have been fixed on the main branch.. but now since you've moved the code to a new place we have to fix them otherwise we will have merge nightmares :D